README.md

```diff
@@ -12,7 +12,7 @@ This bundle provides caching services
 
 Branch | Travis | Coveralls |
 ------ | ------ | --------- |
-3.x   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     |
+2.x   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     |
```

Comment:
@greg0ire this looks wrong to me, do wen need some update on dev-kit?

Reply:
yep, it is wrong, @greg0ire changed this yesterday but dev-kit reverted it 😂

Reply:
Told you it was going to mess things up ^^

Reply:
hehe ok